R-packages/covidcast/R/plot.R

```diff
@@ -272,22 +275,26 @@ plot_bubble = function(x, time_value = NULL, include = c(), range = NULL,
     breaks = seq(lower_bd, range[2], length = num_bins)
   }
 
-  # Create bubble sizes
+  # Max and min bubble sizes
   min_size = params$min_size
   max_size = params$max_size
   if (is.null(min_size)) min_size = ifelse(attributes(x)$geo_type == "county",
                                            0.1, 1)
   if (is.null(max_size)) max_size = ifelse(attributes(x)$geo_type == "county",
                                            4, 12)
```

Comment:
I got the impression these default sizes were carefully tuned to make the map look good by default. Do they need to change now that you're on the area scale (and ignoring the factor of pi)?

Reply:
Good catch!

Reply:
These actually still refer to radii, so the smallest and biggest points should look the same as before.

When I define the sequence of radii to use, I just define them to be equally-spaced on the squared scale, so that the effective areas are equally-spaced on the linear scale.  Check out line 289.

I should be clear: in the code, any reference to "size" (by variable name) still refers to a radius.  Because that's the way `geom_line()` and `scale_size_manual()` treat it.  This is perhaps confusing (and what may have confused you here), because in the documentation, I use "size" (in written language) to refer to an area.